e2e/provision/playbooks/roles/bootstrap/tasks/main.yml

```diff
@@ -34,6 +34,18 @@
   tags:
     - always
 
+- name: Delete docker hub secret
```

Comment:
Suggestion: it might be a good to have L44-L47 added as a separate task under the block at L58. What do you think?

Reply:
Yeah I think that's cleaner, I updated the PR @vjayaramrh